guides/v2.2/config-guide/redis/redis-session.md

```diff
@@ -85,13 +89,14 @@ Magento adds lines similar to the following to `<your Magento install dir>app/et
     ),
 ```
 
+{: .bs-callout .bs-callout-info }
 TTL for session records use the value for Cookie Lifetime, which is configured in Admin. If Cookie Lifetime is set to 0 (the default is 3600), then Redis sessions expire in the number of seconds specified in min_lifetime (the default is 60). This discrepancy is due to differences in how Redis and session cookies interpret a lifetime value of 0. If that behavior is not desired, increase the value of min_lifetime.
 
 ## Basic verification {#redis-verify}
 
 {% include config/redis-verify.md %}
 
-## Related topics
+#### Related topics
```

Comment:
H2 is consistent with other areas of the DevDocs collection. Revert this change.

Reply:
Done please check.